sklearn/ensemble/bagging.py

```diff
@@ -52,6 +53,11 @@ def _parallel_build_estimators(n_estimators, ensemble, X, y, sample_weight,
     bootstrap_features = ensemble.bootstrap_features
     support_sample_weight = has_fit_parameter(ensemble.base_estimator_,
                                               "sample_weight")
+    # Logistic regression does not support sample weights with liblinear
+    if (isinstance(ensemble.base_estimator_,LogisticRegression) and 
```

Comment:
I don't think it is a good idea to put estimator specific parameters here.
With this addition, ``has_fit_paramter`` is not a good way to check if a model supports sample_weights any more.

But do we actually need this case here? I think it's fine to pass the weights in fit and then raise the error from LogisticRegression.